docs/mkdocs.yml

```diff
@@ -91,6 +91,117 @@ nav:
     - how-to/powervs/create-cluster-powervs.md
     - how-to/powervs/create-infra-separately.md
     - how-to/powervs/prerequisites-and-env-guide.md
+- 'Self-Managed Laboratories':
```

Comment:
Thanks for this effort, this is looking great! Can we have this section to be just another category of "how to"? along with AWS and other sections we have there.
Then a subcategory of this would be also "architecture", so all the self managed lab info is self contained there.

Reply:
The problem putting a new section in "how-to-guides" is, once you dropdown on this last section (it's very down bellow) the browser makes you go up because in fact it's a new page, so it's uncomfortable to keep going down every time you get into a new window. This is one of the reasons why I separated this section.

Regarding architecture section, I think it's better to keep it in Reference because it's not related with any Laboratory itself but the organically way to create the environment, it's like reference architectures.